src/components/source_editor/LatticeConfigurationDialog.js

```diff
@@ -61,10 +62,16 @@ class LatticeConfigurationDialog extends React.Component {
         return result;
     }
 
+    /**
+     * Takes in a boolean parameter and uses that to set the value of isDisabled
+     * @param {Boolean} param
+     * @returns {boolean}
+     */
     isDisabled(param) {
+        return param;
         // TODO: implement converter from primitive to conventional cells and re-enable editables
         // const lattice = new Made.Lattice(this.state.lattice);
-        return false // !lattice.editables[param];
+        //return false !lattice.editables[param];
```

Comment:
This PR needs to be cleaned up

Reply:
Do you want me to keep the old comments? I'm not sure if you think they should be saved.